### ISSUE TITLE
RuntimeError: No motif found.

### ISSUE THREAD

**Issue by nikithkurella** [pinellolab/dictys#63]
**Checks before submitting the issue**
* [x] I installed the latest Dictys version successfully
* [x] I downloaded the whole folder for the tutorial
* [x] I ran the tutorial cells and notebooks (if more than one) in the correct order and only once
* [x] The preceding cells and notebooks finished successfully (i.e. the reported error is the first encountered)

**Describe the error**
* How did you install Dictys (which option)?
I am using the prebuilt Docker image
* Which tutorial?
full skin
* What and where was the error?
`RuntimeError: No motif found.` when trying to run `dictys_helper network_inference.sh -j 16 -J 1 dynamic`
* Did you deviate from any of the tutorial instructions?
No
**Optional steps (may accelerate troubleshooting)**
* Attach the error notebook and all preceeding notebooks with run output
I've attached the output to this issue.
[(dictys) root@c696c9102e7c:~:Dictys:full_skin# dic.txt](https://github.com/user-attachments/files/15686256/dictys.root%40c696c9102e7c.Dictys.full_skin.dic.txt)

**Comment by nikithkurella** [comment]
Hi @lingfeiwang. Here it is: 
[main2.txt](https://github.com/user-attachments/files/15686847/main2.txt)

All of the cells up to the dynamic inference step were run in the notebook, but I ran the inference step mentioned previously in terminal.

**Comment by lingfeiwang** [comment]
Can you try reducing the number of jobs? Would `dictys_helper network_inference.sh -j 4 -J 1 dynamic` help?

**Comment by nikithkurella** [comment]
Thanks @lingfeiwang. I will try reducing the number of jobs and report back.

**Comment by nikithkurella** [comment]
I'm receiving the same error even after reducing the number of jobs.

[(dictys) root@c696c9102e7c:~:Dictys:full_skin# dic (2).txt](https://github.com/user-attachments/files/15743911/dictys.root%40c696c9102e7c.Dictys.full_skin.dic.2.txt)

**Comment by lingfeiwang** [comment]
Thank you, nikithkurella.

From the error messages, the most likely reason to me is the Segmentation fault from Homer. We are not familiar with this type of errors so I can only provide several suggestions but none are guaranteed to work.
1. Maybe try `-j 1` instead.
2. Maybe your machine is not 100% compatible with docker or this image generated on github. You can try it on a more typical machine or typical cloud computing machine.
3. You can also try to install Dictys through conda. Following your issue https://github.com/pinellolab/dictys/issues/59, I updated Dictys to be compatible with your machine. Just (re)install the latest version.

If any of them works, please let us know because this is error is absolutely new to us.

Lingfei

**Comment by nikithkurella** [comment]
Hi @lingfeiwang. I've reinstalled the latest version of Dictys through conda on my machine. And I'm now running into a new error `RuntimeError: Boolean value of Tensor with more than one value is ambiguous`. I've attached the full output to this comment.
[dictys.root@c696c9102e7c.Dictys.full_skin.dic (2).txt](https://github.com/user-attachments/files/16058338/dictys.root%40c696c9102e7c.Dictys.full_skin.dic.2.txt)
